src/gui/shell.cpp

```diff
@@ -118,7 +118,7 @@ void Shell::handleFontError(const QString& msg)
 bool Shell::setGuiFont(const QString& fdesc, bool force) noexcept
 {
 	// Exit early if the font description has not changed
-	if (fdesc.compare(fontDesc(), Qt::CaseInsensitive) == 0) {
+	if (!force && (fdesc.compare(fontDesc(), Qt::CaseInsensitive) == 0)) {
```

Comment:
nitpick: no need for the extra parenthesis here

```suggestion
	if (!force && fdesc.compare(fontDesc(), Qt::CaseInsensitive) == 0) {
```